docs/installation/cli-config.rst

```diff
@@ -3,5 +3,29 @@
 Configuring KSQL CLI
 ====================
 
-    
+You can connect the KSQL CLI to one KSQL server per cluster.
+
+.. important:: There is no automatic failover of your CLI session to another KSQL server if the original server that the
+               CLI is connected to becomes unavailable. Any persistent queries you executed will continue to run in the
+               KSQL cluster.
+
+To connect the KSQL CLI to a cluster, run this command with your KSQL Server URL specified (default is ``http://localhost:8088``):
+
+.. code:: bash
+
+    $ <path-to-confluent>/bin/ksql <ksql-server-URL>
+
+
+Configuring Per-session Properties
```

Comment:
Might be worth mentioning that these can be passed along with the `--config-file` argument, or by the `SET` command from within the CLI session.

Reply:
what is `--config-file`? is this something that can be passed to ksql-server? i don't see it doc'd anywhere.